src/hardware/vga_paradise.cpp

```diff
@@ -23,7 +23,7 @@
 #include "inout.h"
 #include "mem.h"
 
-typedef struct {
+typedef struct SVGA_PVGA1A_DATA {
```

Comment:
Remove `typedef` keyword (it's not longer needed now) - this should also clear up new warning that caused build failure on Linux.